build.gradle

```diff
@@ -4,7 +4,7 @@ buildscript {
   ext.versions = [
     kotlin: '1.4.10',
     agp: '3.6.1',
-    layoutlib: '3.4.0', // "should" be similar to versions.agp
+    layoutlib: '4.0.0-alpha01-1a8ff9c', // "should" be similar to versions.agp
```

Comment:
thoughts on versioning our shadowed/relocated layoutlib artifact like so?

I thought that `app.cash.paparazzi:layoutlib:1a8ff9c` was a bit odd...

Reply:
Doesn't seem important. No one sees this and no one is relying on a particular naming scheme. Use whatever is easiest for you or other contributors to understand.

Reply:
I think what you have makes sense, makes it easily identifiable